website/content/docs/connect/intentions.mdx

```diff
@@ -1,12 +1,11 @@
 ---
 layout: docs
-page_title: Service-to-service permissions - Intentions
+page_title: Service Mesh Intentions
 description: >-
-  Intentions define access control for services via Connect and are used to
-  control which services may establish connections or make requests.
+Intentions define communication access in the service mesh through inbound and outbound connection permissions between microservices. Learn about configuration basics, wildcard intentions, precedence and match order, and protecting intention management with ACLs.
```

Comment:
```suggestion
Intentions define communication permissions in the service mesh between microservices. Learn about configuration basics, wildcard intentions, precedence and match order, and protecting intention management with ACLs.
```

Reply:
Should we work "authorization" in here somehow? That seems like a likely search term.